modules/CiteProc/biblio_citeproc.info

```diff
@@ -1,6 +1,8 @@
 name = Biblio - CiteProc
 description = Adds Citation Style Language (CSL) citation processing
-core = 7.x
 package = Biblio
+version = 1.x-2.x
```

Comment:
Version information is added by the packaging script, and should not be included in .info files.